Domain.Api.Tests/TestSetUp.cs

```diff
@@ -0,0 +1,18 @@
+using Microsoft.Its.Domain.Sql;
+
+namespace Microsoft.Its.Domain.Api.Tests
+{
+    public static class TestSetUp
+    {
+        public static void InitializeEventStore()
+        {
```

Comment:
This is a fairly expensive operation and doesn't need to happen more than once per test run, so renaming to `EnsureEventStoreIsInitialized` and returning early if initialization has already happened would be preferable. Another option is to do `Database.SetInitializer<EventStoreDatabaseInitializer<EventStoreDbContext>>()` which will initialize the database lazily when `EventStoreDbContext` is first instantiated.